src/components/menu/examples/menu-open-direction.tsx

```diff
@@ -0,0 +1,82 @@
+import {
+    MenuItem,
+    ListSeparator,
+    OpenDirection,
+    Option,
+} from '@limetech/lime-elements';
+import { Component, h, State } from '@stencil/core';
+
+/**
+ * openDirection
+ *
+ * The value of the `openDirection` property defines how the menu is aligned,
+ * and in which direction it extends.
+ */
+@Component({
+    tag: 'limel-example-menu-open-direction',
+    shadow: true,
+})
+export class MenuOpenDirectionExample {
+    @State()
+    private selectedOpenDirection: Option<OpenDirection>;
+
+    private availableOpenDirections: Array<Option<OpenDirection>>;
+
+    private items: Array<MenuItem | ListSeparator> = [
+        { text: 'Copy' },
+        { text: 'Cut' },
+        { separator: true },
+        { text: 'Paste' },
+    ];
+
+    constructor() {
+        const openDirections: OpenDirection[] = [
+            'left-start',
+            'left',
+            'left-end',
+            'right-start',
+            'right',
+            'right-end',
+            'top-start',
+            'top',
+            'top-end',
+            'bottom-start',
+            'bottom',
+            'bottom-end',
+        ];
+        this.availableOpenDirections = openDirections.map((value) => {
+            return {
+                text: value as string,
+                value: value,
+            } as Option<OpenDirection>;
+        });
+        this.selectedOpenDirection = this.availableOpenDirections.find(
+            (v) => v.value === 'bottom-start'
+        );
+    }
+
+    public render() {
+        return [
+            <p>
+                <limel-menu
+                    items={this.items}
+                    openDirection={this.selectedOpenDirection?.value}
+                >
+                    <limel-button label="Menu" slot="trigger" />
+                </limel-menu>
+            </p>,
+            <limel-select
```

Comment:
I'll just make the UI a bit nicer here, and push soon 
<img width="722" alt="image" src="https://user-images.githubusercontent.com/35954987/178208568-51f2c064-efe1-4b97-9ccf-fca88bf32e03.png">

Reply:
⚡️ Done
<img width="494" alt="image" src="https://user-images.githubusercontent.com/35954987/178211535-698ca1c5-ecb0-4f9e-ad68-a1c9ccb605dc.png">

Reply:
Very nice!

I'm adding a small fixup, which sets a min-width, to avoid this problem:

<img width="324" alt="image" src="https://user-images.githubusercontent.com/154725/178218764-00eb5e4f-1bad-4732-a281-8a4d54e7a469.png">

Reply:
Here's a screenshot of the fixup diff:

<img width="716" alt="Screenshot showing that `limel-select { min-width: 8.75rem; }` has been added to `src/components/menu/examples/menu-open-direction.scss`." src="https://user-images.githubusercontent.com/154725/178219328-8d537e8d-4cdf-432e-84e7-63a0954c7cb8.png">

I'll go ahead and squash it, since it's so small and I'm about to squash the older fixups now anyway.